code/style.css

```diff
@@ -1,4 +1,401 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
 html {
-  background: #0025ff;
-  color: #fff;
+  background: white;
+  color: white;
+  font-family: 'Montserrat', sans-serif;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+/* START Header with logo, links and date only visible on mobile (max 1024px) */
+header.mobile {
+  height: 4.5rem;
+  border: solid #82171A;
+  border-width: 0.5rem 0;
+  margin-bottom: 1rem;
+  padding: 0 0.5rem;
+}
+
+.mobile-wrap {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  z-index: 2;
+}
+
+.mobile-wrap-item1 {
+  width: 8rem;
+  text-align: left;
+}
+
+.mobile-wrap-item2 {
+  width: 8rem;
+  text-align: center;
+}
+
+.mobile-wrap-item3 {
+  width: 8rem;
+  text-align: right;
+}
+
+.logo-mobile {
+  height: 3rem;
+  padding-top: 0.5rem;
+}
+
+.menu-icon {
+  font-size: 2rem;
+  padding-top: 0.5rem;
+  color: #82171A;
+  line-height: 2rem;
+}
+
+.mobile-wrap nav {
+  position: absolute;
+  background-color: #82171A;
+  top: 4.5rem;
+  left: 0;
+  width: 50%;
+  z-index: 1;
+}
+
+.mobile-wrap nav ul {
+  list-style-type: none;
+  max-height: 0;
+  overflow: hidden;
+  transition: all 0.5s;
+}
+
+.mobile-wrap nav ul li {
+  padding: 0 0.5rem;
+}
+
+.mobile-wrap nav ul li a {
+  display: inline-block;
+  padding: 0.75rem;
+  color: white;
+  text-decoration: none;
+}
+
+#checkbox {
+  display: none;
+}
+
+#checkbox:checked~nav ul {
+  max-height: 18rem;
+  padding: 1rem 0;
+  transition: all 0.5s;
+}
+
+@media (min-width: 1025px) {
+  header.mobile {
+    display: none;
+  }
+}
+/* END Header with logo, links and date only visible on mobile (max 1024px) */
+
+/* START Header with logo, links and date only visible on desktop (min 1025px) */
```

Comment:
I like that you have commented on all your code with start and end of every style, so it´s easy to understand. In this part, it´s really specified what screen size you have coded for.